sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/CheckRestrictedAreas.java

```diff
@@ -0,0 +1,178 @@
+            throw new SiddhiAppValidationException("Invalid parameter type found for third argument " +
+                    "current.login.country of geovelocity:loginbehaviourbasedrisk() function, required "
+                    + Attribute.Type.STRING + ", but found " + thirdattributeType.toString());
+        }
+        Attribute.Type fourthattributeType = attributeExpressionExecutors[3].getReturnType();
+        if (fourthattributeType != Attribute.Type.STRING) {
+            throw new SiddhiAppValidationException("Invalid parameter type found for fourth argument " +
+                    "previous.login.country of geovelocity:loginbehaviourbasedrisk() function, required "
+                    + Attribute.Type.STRING + ", but found " + fourthattributeType.toString());
+        }
+    }
+
+    @Override
+    protected Object execute(Object[] data) {
+
+        GeoVelocityData geoVelocityData;
+        geoVelocityData = geoVelocityDataResolverImpl.checkLoginLocationValidity
+                (data[0].toString(), data[1].toString(), data[2].toString(), data[3].toString());
+        return geoVelocityData.checkSuspiciousLogin();
+    }
+
+    @Override
+    protected Object execute(Object data) {
+        return null;
+    }
+
+    @Override
+    public Attribute.Type getReturnType() {
+        return Attribute.Type.INT;
+    }
+
+    @Override
+    public Map<String, Object> currentState() {
+        return null;
+    }
+
+    @Override
+    public void restoreState(Map<String, Object> state) {
+    }
+
+    private void initializeExtensionConfigs(ConfigReader configReader) throws SiddhiAppValidationException {
+
+        String geovelocityResolverImplClassName = configReader.readConfig("geoVelocityResolverClass",
+                DEFAULT_GEOVELOCITY_RESOLVER_CLASSNAME);
+        try {
+            geoVelocityDataResolverImpl = GeoVelocityDataResolverHolder.getGeoVelocityResolverHolderInstance().
+                    getGeoVelocityDataResolver(geovelocityResolverImplClassName);
+            geoVelocityDataResolverImpl.init(configReader);
+        } catch (GeoVelocityException e) {
+            throw new SiddhiAppValidationException("Cannot initialize " +
+                    "GeoVelocityDataResolver implementation class '"
+                    + geovelocityResolverImplClassName + "' given in the configuration", e);
+        } catch (InstantiationException e) {
+            throw new SiddhiAppValidationException("Cannot instantiate GeoCoordinateResolverHolder holder class '"
+                    + geovelocityResolverImplClassName , e);
+        } catch (IllegalAccessException e) {
+            throw new SiddhiAppValidationException("Cannot access GeoCoordinateResolverHolder holder class '"
+                    + geovelocityResolverImplClassName , e);
+        } catch (ClassNotFoundException e) {
+            throw new SiddhiAppValidationException("Cannot find GeoCoordinateResolverHolder holder class '"
```

Comment:
The closing single quotation mark is missing.

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265